testsuite/unittests/test_gap_junction.sli

```diff
@@ -82,7 +98,7 @@ neuron_no_gap neuron_no_gap Connect
 0 /max_delay get /old_max Set
 0 /min_delay get /old_min Set
 
-neuron_gap neuron_gap 4.0 4.0 /gap_junction Connect
+neuron_no_gap neuron_no_gap 2.0 2.0 Connect
```

Comment:
@janhahne Would `neuron_gap neuron_gap /gap_junction Connect` the proper code here? With the `4.0 4.0` in addition it must be incorrect, since one cannot set delay for gap junctions, or?

Note that you here have "naked" code in a test script (not under any `*_or_die` control). That is best avoided. Could you put it into a setup function and then call that from the individual tests? That ensures that tests are independent. If tests are naturally dependent, you can also test different things within a single `assert_or_die` and then just `and` the results of the different parts of the test.